net/qpid-proton/Portfile

```diff
@@ -1,7 +1,9 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
+github.setup        apache qpid-proton 0.31.0
+revision            0
+
```

Comment:
@mf2k Was the very top the right place under the formatting comment?

Reply:
It should go directly after the `PortSystem          1.0` line.

Reply:
And also after adding the GitHub portgroup line.